.circleci/config.yml

```diff
@@ -25,30 +25,33 @@ runOnDocker: &runOnDocker
   working_directory: /go/src/github.com/mesg-foundation/core
 
 jobs:
-  "build_docker":
-    <<: *runOnMachine
+  "unit_test":
+    <<: *runOnDocker
     steps:
       - checkout
-      - run: docker pull mesg/core:latest
-      - run: docker build -t mesg/core:$CIRCLE_SHA1 --build-arg version="dev build `echo $CIRCLE_SHA1 | cut -c1-7`" .
-
-  "test":
+      - setup_remote_docker # this should be remove when no test are actually using docker
+      - run: go get -t ./...
+      - run: docker swarm init # this should be remove when no test are actually using docker
+      - run: docker build -t sleep docker-images/sleep/ # this should be remove when no test are actually using docker
+      - run: docker build -t http-server docker-images/http-server/ # this should be remove when no test are actually using docker
+      - run: env MESG_CORE_IMAGE=mesg/core:NONEXISTINGIMAGE go test -timeout 180s -p 1 -coverprofile=coverage.txt ./...
```

Comment:
What about syntax:
```
run:
  name:
  command
  envrioment:
```
I think is more clear than putting everything in one line

Reply:
We don't have the env variable anymore and CircleCi doesn't provide to anyway to pass arguments through a nice yaml list.
I'm ok to read the command and not a short name.
What do you think @krhubert?